src/components/flow/actions/callbrain/CallBrainForm.tsx

```diff
@@ -8,21 +9,42 @@ import * as React from 'react';
 import styles from './CallBrainForm.module.scss';
 import i18n from 'config/i18n';
 import AppState from 'store/state';
-import { propsToAction } from './helpers';
+import { updateAction, initializeForm } from './helpers';
 import { BrainInfo } from '../../../../store/flowContext';
 
 import { applyVueInReact } from 'veaury';
 // @ts-ignore
 import Unnnic from '@weni/unnnic-system';
+import TextInputElement, {
+  TextInputSizes,
+} from 'components/form/textinput/TextInputElement';
+import { StringEntry } from 'store/nodeEditor';
 const UnnnicIcon = applyVueInReact(Unnnic.unnnicIcon);
 
 export interface CallBrainFormProps extends ActionFormProps {
   brainInfo: BrainInfo;
+  entry: string;
 }
 
-export class BrainForm extends React.Component<CallBrainFormProps> {
+export interface CallBrainFormState {
+  entry: StringEntry;
+}
+
+export class BrainForm extends React.Component<
+  CallBrainFormProps,
+  CallBrainFormState
+> {
+  constructor(props: CallBrainFormProps) {
+    super(props);
+    this.state = initializeForm(this.props.nodeSettings);
+    bindCallbacks(this, {
+      include: [/^handle/, /^on/],
+    });
+  }
   private handleSave(): void {
-    this.props.updateAction(propsToAction(this.props.nodeSettings, this.props));
+    const brainState = { ...this.props };
+    brainState.entry = this.state.entry.value;
```

Comment:
Just to ensure the data is being saved and sent correctly, use the `trim()` method to remove and trailing spaces